### PR TITLE
delete all targets without target parameter

### PR DESCRIPTION
**Pull request by pivotal-bin-ju**
PR for https://github.com/concourse/concourse/issues/4114

### CODE REVIEW

**Comment by deniseyu** [comment]
I added the test case, and also noticed that we didn't have a test case for when no target is specified. hope that's ok @pivotal-bin-ju!

**Comment by jomsie** [comment]
i saw wats failed, probably because of the recent watsjs problems on the ui, so i restarted it with this pr

File: fly/integration/delete_target_test.go [hunk 1]
```diff
@@ -74,6 +74,20 @@ var _ = Describe("Fly CLI", func() {
 				os.RemoveAll(tmpDir)
 			})
 
+			Context("when no fly target is specified", func() {
+				It("asks the user to specify a pipeline", func() {
```
Comment (vito) [inline fly/integration/delete_target_test.go]:
```suggestion
				It("asks the user to specify a target or log in", func() {
```

**Comment by xtremerui** [comment]
@vito could you review this one again?

**Comment by vito** [comment]
Gonna close this out since it looks like the commit history got a little borked. Feel free to resubmit!

**Pull request closed without merging**